Number_types/include/CGAL/GMP/Gmpzf_type.h

```diff
@@ -422,7 +422,7 @@ std::pair<std::pair<double, double>, long> Gmpzf::to_interval_exp() const
   // get surrounding interval of the form [l * 2 ^ k, u * 2^ k]
   // first get mantissa in the form l*2^k, with 0.5 <= d < 1;
   // truncation is guaranteed to go towards zero
-  long k = 0;
+  Exponent k = 0;
   double l = mpz_get_d_2exp (&k, man());
```

Comment:
https://github.com/wbhart/mpir/blob/master/gmp-h.in#L918 seems to say that `mpz_get_d_2exp` also expects a `long` in MPIR now. If we change things to help older versions of MPIR, that's going to break future ones...